comfy_cli/command/generate/app.py

```
@@ -0,0 +1,370 @@
+            rprint(
+                "[yellow]Binary image response; nothing saved. "
+                "Pass --download <path> to write it to disk.[/yellow]"
+            )
+        return
+
+    try:
+        body = resp.json()
+    except ValueError:
+        rprint("[bold red]Unexpected non-JSON response.[/bold red]")
+        rprint(resp.text[:500])
+        raise typer.Exit(code=1)
```

Pylint (4.0.5)

[warning] Consider explicitly re-raising using 'except ValueError as exc' and 'raise typer.Exit(code=1) from exc'

(W0707)